### PR TITLE
First pass adding tx env vars

### PR DESCRIPTION
**Pull request by alexanderattar**
Adds the env vars required to enable deposits / withdrawals. The build in GA currently fails because the tests bomb out when there are these missing variables.

### CODE REVIEW

File: docker-compose.env [hunk 1]
```diff
@@ -59,3 +59,12 @@ RUN_STATE_BATCH_SUBMITTER=true
...
 L2_NODE_WEB3_URL=http://geth_l2:8545
+
+######### L1->L2 / L2->L1 Tests #########
+L1_USER_PRIVATE_KEY="0x754fde3f5e60ef2c7649061e06957c29017fe21032a8017132c0078e37f6193a"
+L2_USER_PRIVATE_KEY="0x570afe67c36ac0ae058df4b5c28b49e1381ffbce761866750f0cc41918d0d383"
+L1_MESSENGER_ADDRESS="0x6418E5Da52A3d7543d393ADD3Fa98B0795d27736"
+L2_MESSENGER_ADDRESS="0x4200000000000000000000000000000000000007"
+
+# Address Manager
+ETH1_ADDRESS_RESOLVER_ADDRESS=0x3e4CFaa8730092552d9425575E49bB542e329981
```
Comment (karlfloersch) [inline docker-compose.env:70]: This is already supplied via https://github.com/ethereum-optimism/docker/blob/f0db880135371ea61cf747ee9de6062f20ea99f5/integration-tests/wait-for-l1-and-l2-and-contract-deployment.sh#L55

File: docker-compose.env [hunk 2]
```diff
@@ -59,3 +59,12 @@ RUN_STATE_BATCH_SUBMITTER=true
 
 L1_NODE_WEB3_URL=http://l1_chain:9545
 L2_NODE_WEB3_URL=http://geth_l2:8545
+
+######### L1->L2 / L2->L1 Tests #########
+L1_USER_PRIVATE_KEY="0x754fde3f5e60ef2c7649061e06957c29017fe21032a8017132c0078e37f6193a"
+L2_USER_PRIVATE_KEY="0x570afe67c36ac0ae058df4b5c28b49e1381ffbce761866750f0cc41918d0d383"
+L1_MESSENGER_ADDRESS="0x6418E5Da52A3d7543d393ADD3Fa98B0795d27736"
+L2_MESSENGER_ADDRESS="0x4200000000000000000000000000000000000007"
```
Comment (karlfloersch) [inline docker-compose.env:67]: We can hardcode this (for now)  -- it should never change

File: docker-compose.env [hunk 3]
```diff
@@ -59,3 +59,12 @@ RUN_STATE_BATCH_SUBMITTER=true
 
 L1_NODE_WEB3_URL=http://l1_chain:9545
 L2_NODE_WEB3_URL=http://geth_l2:8545
+
+######### L1->L2 / L2->L1 Tests #########
+L1_USER_PRIVATE_KEY="0x754fde3f5e60ef2c7649061e06957c29017fe21032a8017132c0078e37f6193a"
+L2_USER_PRIVATE_KEY="0x570afe67c36ac0ae058df4b5c28b49e1381ffbce761866750f0cc41918d0d383"
+L1_MESSENGER_ADDRESS="0x6418E5Da52A3d7543d393ADD3Fa98B0795d27736"
```
Comment (karlfloersch) [inline docker-compose.env:66]: Pull from AddressManager

Comment (karlfloersch) [reply docker-compose.env:66]: (aka address resolver)

**Comment by K-Ho** [comment]
closing after chat! should be able to get these things without new env vars!

**Pull request closed without merging**